### PR TITLE
[271] GtR general ES pipeline

### LINKED ISSUE #271: [DAPS] Request validation on GtR data
then run in production mode, then add to daily schedule

### PR DESCRIPTION
**Pull request by jaklinger**
Closes #271 

- [x] Write batchable
- [x] Write routine
- [x] Write tests (nesta/core/batchables/general/gtr/tests/test_gtr.py)
- [x] Run `dev` pipeline + iterate
- [x] Run `production` pipeline + iterate
- [x] Request feedback + iterate

### CODE REVIEW

**Comment by jaklinger** [comment]
Opened #289 which explains why GtR outcomes aren't fully included. Instead, only a count of outcomes by outcome type is provided.

**Comment by jaklinger** [comment]
@saabi @mindrones I'll ping the ES endpoint for this. Note this is ES7 for GtR data.

@bishax if you could review the python side of things!

File: nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json [hunk 1]
```diff
@@ -23,9 +17,11 @@
                         "type": "keyword"
                     },
                     "end_date": {
+                        "format": "strict_date_hour_minute_second",
                         "type": "date"
                     },
                     "start_date": {
+                        "format": "strict_date_hour_minute_second",
                         "type": "date"
```
Comment (mindrones) [inline nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json]: I'm wondering if/why we need funding dates with the second precision, if there are no particular reasons I'd convert these dates to `YYYY-MM-DD`.
I'd also propose to do this for for all of our dates where applicable.

**Review by mindrones — APPROVED** [review]
Good to go on my part, but possibly I'd address the comment below about dates.

File: nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json (hunk 1 above)
Comment (jaklinger) [reply nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json]: Yeah, that's fair. I'll convert to `strict_date` which is `YYYY-MM-DD`

**Pull request merged**